README.md

```diff
@@ -123,10 +132,106 @@ To install pgai from source on a PostgreSQL server:
    The `CASCADE` automatically installs `pgvector` and `plpython3u` extensions.
 
 ### Use pgai
- 
-Now, use pgai to integrate AI from [Ollama](./docs/ollama.md), [OpenAI](./docs/openai.md), 
-[Anthropic](./docs/anthropic.md), and [Cohere](./docs/cohere.md). Learn how to [moderate](./docs/moderate.md) and
-[embed](./docs/delayed_embed.md) content directly in the database using triggers and background jobs.
+
+The main features in pgai are:
+
+**Working with embeddings generated from your data:**
+* [Automatically create and sync vector embeddings for your data](#automatically-create-and-sync-llm-embeddings-for-your-data)
+* [Search your data using vector and semantic search](#search-your-data-using-vector-and-semantic-search)
+* [Implement Retrieval Augmented Generation inside a single SQL statement](#implement-retrieval-augmented-generation-inside-a-single-sql-statement) 
+
+**Leverage LLMs for data processing tasks:**
+You can use pgai to integrate AI from [Ollama](./docs/ollama.md), [OpenAI](./docs/openai.md), 
+[Anthropic](./docs/anthropic.md), and [Cohere](./docs/cohere.md). Learn how to [moderate](./docs/moderate.md) and content directly in the database using triggers and background jobs.
+
+
+### Automatically create and sync LLM embeddings for your data
+
+The [pgvector](https://github.com/pgvector/pgvector) and 
+[pgvectorscale](https://github.com/timescale/pgvectorscale) extensions allow you
+to store vector embeddings in your database and perform fast and efficient
+vector search.  The [pgai Vectorizer](/docs/vectorizer.md) builds on top of
+these extensions to automatically create and synchronizes embeddings for any
+text data in your database.
+
+With one line of code, you can define a vectorizer that creates embeddings for data in a table:
+```sql
+SELECT ai.create_vectorizer( 
+    <table_name>::regclass,
+    destination => <embedding_table_name>,
+    embedding => ai.embedding_openai(<model_name>, <dimensions>),
+    chunking => ai.chunking_recursive_character_text_splitter(<column_name>)
+);
+```
+This newly created vectorizer will automatically track any changes to the
+data in the source table and update the destination embedding table
+with the new embeddings asynchronously.
+
+[Automate AI embedding with pgai Vectorizer](/docs/vectorizer.md) shows you how
+to implement embeddings in your own data. When you install pgai on Timescale
+Cloud, embedding are automatically created and synchronized in the background.
+On a self-hosted Postgres installation, you use a [Vectorizer
+Worker](/docs/vectorizer-worker.md) to asynchronously processes your
+vectorizers.
+
+### Search your data using vector and semantic search
+
+pgai exposes a set of functions to directly interact with the LLM models through SQL, enabling 
+you to do semantic search directly in your database:
+
+```sql
+SELECT 
+   chunk,
+   embedding <=> ai.openai_embed(<embedding_model>, 'some-query') as distance
+FROM <embedding_table>
+ORDER BY distance
+LIMIT 5;
+```
```

Comment:
Should we replace this with a real query and also show result data? If you scroll through to understand how it works it might be helpful to see what such a query accomplishes.

Reply:
Good idea. Can I leave this to you in a followup pr?

Reply:
sure 👍